source/game/field/obj/objects/Mdush.hh

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "game/field/obj/GeoObject.hh"
+#include <egg/math/Matrix.hh>
+
+namespace Field
+{
+
+class Mdush : public GeoObjectKCL {
```

Comment:
@vabold Do we want to add `final` keyword on the inheritance?

Reply:
Yes.